### PR TITLE
Automated accessibility testing AR-1224

### LINKED ISSUE #1019: Action to calculate accessibility rating in CI
Issue raised by: developer

### What?
Add a CI job that calculates the accessibility rating/finds accessibility issues of the webapp.

### PR DESCRIPTION
**Pull request by rickybassom**
### Summary of work
* Uses `axe-selenium-python` for Selenium accessibility testing
* Automated in GitHub actions
* A report is generated which is put into the GitHub Actions artefact
* Also enables ability to exclude accessibility rules if needed
* Adds tests for all pages:
  - [x] Overview page
  - [x] Configure new runs page
  - [x] Rerun jobs page
  - [x] Run summary page
  - [x] Runs list page
  - [x] Variables summary page
  - [x] Help page
* Fixed newly found accessibility problems or opened issues 

### How to test your work
* Tests pass
* Code review

Fixes #1019

**Before merging ensure the release notes have been updated**

### CODE REVIEW

**Comment by rickybassom** [comment]
I can't seem to disable duplicate code pylint warnings https://github.com/PyCQA/pylint/issues/214

File: WebApp/autoreduce_webapp/selenium_tests/tests/base_tests.py [hunk 1]
```diff
@@ -44,9 +45,10 @@ def tearDown(self) -> None:
 
     def _screenshot_driver(self):
         now = datetime.datetime.now()
-        screenshot_name = f"{self._testMethodName}-{now.strftime('%Y-%m-%d_%H-%M-%S')}.png"
-        path = str(Path(PROJECT_ROOT, "WebApp", "autoreduce_webapp", "selenium_tests", "screenshots", screenshot_name))
-        self.driver.save_screenshot(path)
+        screenshot_name = f"{type(self).__name__}-{self._testMethodName}-{now.strftime('%Y-%m-%d_%H-%M-%S')}.png"
+        report_path = str(
```
Comment (keiranjprice101) [inline WebApp/autoreduce_webapp/selenium_tests/tests/base_tests.py:49]: why report path?

**Comment by keiranjprice101** [comment]
To deal with the duplicates would it be possible to have the shared ignore on the `Mixin` and then if new rule is needed per page call something like `base_rules..update(rules)` ?

File: WebApp/autoreduce_webapp/selenium_tests/tests/base_tests.py (hunk 1 above)
Comment (rickybassom) [reply WebApp/autoreduce_webapp/selenium_tests/tests/base_tests.py:49]: Didn't see that. I'll change it to `screenshot_path`

**Comment by rickybassom** [comment]
Change based to the release because of fixed a11y issues

**Comment by rickybassom** [comment]
Demo of a failed a11y test with GitHub actions artefacts `bfac15a`

**Comment by keiranjprice101** [comment]
I have looked through a couple of files in the artefact and they are very very long. Is there not a way to generate something that is more human readable? Though the fail message is nice

**Comment by rickybassom** [comment]
> I have looked through a couple of files in the artefact and they are very very long. Is there not a way to generate something that is more human readable? Though the fail message is nice

I've just added an option to change the kind of report axe produces. Default is set to just report violations.

**Pull request merged**